modules/images/webp-uploads/fallback.js

```diff
@@ -0,0 +1,73 @@
+( function( d ) {
```

Comment:
```suggestion
( function( document ) {
```

As this is the full version of the script, I'd pass the document as `document` and the advantage of compression can be handled by the build process. 

Using `d` like this often results in future developers adding code with `document` and then the advantage of passing the variable is lost. 

:1234: would require changes throughout

Reply:
Good point. Updated.